### PR TITLE
Use css variables for colors and anything is reusable on sass files #826

### PR DESCRIPTION
**Pull request by priscilawebdev**
I created sass variables for css that could be reusable and also mixins. As we will discuss about a possible implementation of css in js, I think it is not a good idea to spend a lot of time refactoring sass...Feedbacks and suggestions are welcome.

### CODE REVIEW

**Comment by juanpicado** [comment]
@priscilawebdev commit message should be `@priscilawebdev
refactor: xxx`

File: src/webui/styles/variables.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
+// Verdaccio
+// -------------------------
+
+$black:         #000;
```
Comment (juanpicado) [inline src/webui/styles/variables.scss]: I like this so much 👍

**Comment by priscilawebdev** [comment]
@juanpicado fixed both 👍

**Comment by juanpicado** [comment]
Great contribution!! 👩🏼‍💻👌🏼

**Pull request merged**